jwt.el

```diff
@@ -613,13 +613,58 @@ Specifically it checks that TEST-STRING has
            (doco (car maybe-doc)))
       (funcall callback doco :thing full-name))))
 
+(defvar jwt--annotation-claims
+  '(("nbf" . "Not before")
+    ("iat" . "Issued at")
+    ("exp" . "Expires")))
+
+(defface jwt-annotation
+  '((t
+     (:box
+      (:line-width (1 . 1)
+                   :color nil
+                   :style nil)
+      :inherit (magit-branch-local))))
+  "Face for JWT claim annotation overlays.")
+
+(defun jwt--annotation-add-overlays (beg end)
+  "Add JWT related overlays between BEG and END."
+  (save-excursion
+    (goto-char beg)
+    (let ((claims-rx (rx "\"" (eval (cons 'group (list (cons 'or (mapcar #'car jwt--annotation-claims))))) "\""
```

Comment:
You can simplify the first group in the `rx` sexp by pushing the `eval` to the inner sexp:
```elisp
(group (or (eval (mapcar #'car jwt--annotation-claims))))
```
You can also use the `map-keys` function to get the alist keys, make sure to add `(require 'map)` if you do.